src/pages/ListContainer/ListContainer.tsx

```diff
@@ -57,6 +67,34 @@ export const ListContainer: React.FC<ListTypes> = ({ list, setList }) => {
           />
         )}
       </div>
+      <div>
+        <CSVReader
+          onUploadAccepted={(results: any) => {
+            if (results.errors.length) {
+              alert('faildToAddCount');
+            } else {
+              const users = results.data.map((item: UserInfo[]) => {
+                return {
+                  id: item[0],
+                  firstName: item[1],
+                  lastName: item[2],
+                  age: item[3],
+                  country: item[4],
+                  gender: item[5],
+                };
+              });
+              users.shift();
+              setFilteredUsers(users);
+              alert('addedCount');
+            }
+          }}>
+          {({ getRootProps }: any) => (
+            <button type="button" {...getRootProps()} className={styles.button}>
+              Import Contacts
+            </button>
+          )}
+        </CSVReader>
+      </div>
```

Comment:
Why in this PR are old changes? that should be merged and new branch should have the last sources